packages/realm-react/example/android/app/build.gradle

```diff
@@ -120,6 +120,11 @@ def jscFlavor = 'org.webkit:android-jsc:+'
  */
 def enableHermes = project.ext.react.get("enableHermes", false);
 
+/**
+ * Architectures to build native code for in debug.
+ */
+def nativeArchitectures = project.getProperties().get("reactNativeDebugArchitectures")
```

Comment:
Are these changes a result of us standardising on a single version of dependencies or something else? Also are the various lint fixes a result of us unifying our lint configs?

Reply:
These are simply from upgrading the React Native version of the realm-react/example app. I wanted to align on a single version of React Native across the sub-packages. Strictly that might not have been needed, I could revert if this is an issue.